daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -905,19 +905,17 @@ private[lf] object SBuiltin {
     *    -> Bool (false if ensure failed)
     *    -> Unit
     */
-  final case class SBCheckPrecond(templateId: TypeConName) extends SBuiltinPure(3) {
+  final case class SBCheckPrecond(templateId: TypeConName) extends SBuiltinPure(2) {
```

Comment:
Please update the comment above (github won't let me make a diff suggestion since the top line is "too far" from your changes)

```diff
   /** $checkPrecondition
     *    :: arg (template argument)
-    *    -> Unit
     *    -> Bool (false if ensure failed)
     *    -> Unit
     */
```

Reply:
Good catch